aiokafka/consumer.py

```diff
@@ -191,6 +192,7 @@ def __init__(self, *topics, loop,
     @asyncio.coroutine
     def start(self):
         yield from self._client.bootstrap()
+        yield from self._wait_topics()
```

Comment:
Why do we do it before we start coordinator and fetcher? This will sadly not fix:
```python
consumer = await AIOKafkaConsumer(bootstrap_servers="localhost:9092")
consumer.start()
consumer.subscribe("some_autocreate_topic")
```